app/2.0/start/toolbox/deploy.md

```diff
@@ -143,10 +162,33 @@ guide](https://www.firebase.com/docs/hosting/quickstart.html).
     ]
     ```
 
+    For example, your `firebase.json` file may look like this afterwards:
+	
+    ```
+    {
+      "database": {
+        "rules": "database.rules.json"
+      },
+      "hosting": {
+        "public": "build/default",
+        "rewrites": [
+          {
+            "source": "!/__/**",
+            "destination": "/index.html"
+          },
+          {
+            "source": "**/!(*.js|*.html|*.css|*.json|*.svg|*.png|*.jpg|*.jpeg)",
```

Comment:
Could you clarify what you mean? 

Do you mean change 

`{+            "source": "**/!(*.js|*.html|*.css|*.json|*.svg|*.png|*.jpg|*.jpeg)", `

to 

`{+            "source": "**/!(*.*)",`

?

I can definitely do that if that's what you mean.